string[] joint_names
float64[] command
